### ISSUE TITLE
Validate Parts/Fields in Handler instead of  Display

### ISSUE THREAD

**Issue by ns8482e** [OrchardCMS/OrchardCore#6147]
Currently validation is handled in Display Driver hence, content can only be validated from Admin UI. If validation rule is defined in Content Definition.  It should be validated before saving.

**Comment by sebastienros** [comment]
UI validation might need to do more things that generic validation, because it will validate user inputs instead of model values. The new Validating event from handlers might be the way to validate the data. There is a method in the content manager to call that.

c.f. https://github.com/OrchardCMS/OrchardCore/pull/6195

**Comment by ns8482e** [comment]
Nice to have Field Handler having Validating event too

**Comment by Skrypt** [comment]
Yeah I agree that the validation in the drivers should be moved into a common method that everything uses.
